nav2_costmap_2d/plugins/static_layer.cpp

```diff
@@ -72,7 +72,10 @@ StaticLayer::onInitialize()
   global_frame_ = layered_costmap_->getGlobalFrameID();
 
   getParameters();
-  getMap();
+
+  if (first_map_from_service_) {
```

Comment:
@bpwilcox This opens up the possibility that we can get through static layer initialization without having any map data. Is that safe?

Reply:
Unclear for now, I'll look into this.

Reply:
You are right. I made sure to initialize it with empty map data, which makes the other layer plugins happy enough to not crash